src/main/java/TaskList.java

```diff
@@ -0,0 +1,72 @@
+import java.awt.*;
+import java.util.ArrayList;
+
+public class TaskList {
+
+    private final ArrayList<Task> list;
+
+    public TaskList(){
+        list = new ArrayList<>();
+    }
+
+    public String add_task(Task task) throws TaskListFullException {
+
+        if ( list.size() >= 100) {
+            throw new TaskListFullException("Sorry, the task list is full.");
+        }
```

Comment:
Consider to remove this as you are using the `ArrayList<Task>`. This can allow user to store more task in the list.